SocialFilter/sections/specific.txt

```diff
@@ -45,6 +45,8 @@ duzcedamla.com,muglagazetesi.com.tr,apron24.com,bilecikses.com,enzahome.com.tr,d
 apusapps.com##.share-layer
 trapz.io##.youtuber
 trapz.io###gameover > div.main-trapz__form-wrapper + div.main-trapz__download[style="margin-top: 43px;"]
+4shared.com##.jsFBsharingLink
+4shared.com##.fbsharing
```

Comment:
Not shown.

Reply:
![Screenshot 2020-07-24 at 12 47 38](https://user-images.githubusercontent.com/18536328/88379623-f878f600-cdab-11ea-91d9-857d4dc917c5.png)